engineapi/engineapi/routes/metatx.py

```diff
@@ -37,7 +37,9 @@
 whitelist_paths = {
     "/metatx/openapi.json": "GET",
     f"/metatx/{DOCS_TARGET_PATH}": "GET",
+    "/metatx/blockchains": "GET",
     "/metatx/contracts/types": "GET",
```

Comment:
We should remove `metatx/contracts/types`?

Reply:
Saved for back capability, can delete it